todo_app/data/trello_items.py

```diff
@@ -0,0 +1,70 @@
+from todo_app.data.item import Item
+import requests
+from os import getenv
+
+board_id = getenv('BOARD_ID')
+api_key = getenv("API_KEY")
+token = getenv("TOKEN")
+auth_params = f'key={api_key}&token={token}'
+
+board_lists = requests.get(f'https://api.trello.com/1/boards/{board_id}/lists?fields=name&{auth_params}').json()
+for board_list in board_lists:
+    if board_list['name'] == "To Do":
+        to_do_list_id = board_list['id']
+    elif board_list['name'] == "Done":
+        done_list_id = board_list['id']
+
+list_names = {to_do_list_id: "To Do", done_list_id: "Done"}
+
+
+def get_items():
+    cards = requests.get(f'https://api.trello.com/1/boards/{board_id}/cards?fields=name,idList&{auth_params}').json()
```

Comment:
You have repeated the start of this URL 5 times in this file. Repeating yourself make the code less maintainable.

Please can you extract a function called `build_url` which accepts the unique part of the URL and prepends the shared part?